test/cypress/e2e/oidc/oidc_auth_test.spec.js

```diff
@@ -115,8 +115,12 @@ describe('Log in via OIDC', () => {
 
     cy.get('button[id="user-icon-btn"]').click();
 
+    cy.intercept('GET', `${basePath}/auth/openid/logout`).as('openidLogout');
+
     cy.get('button[data-test-subj^="log-out-"]').click();
 
+    cy.wait('@openidLogout').then(() => {});
+
```

Comment:
I was having issues with the `afterEach` hook that turned out to be related to not including `opensearch_security.openid.base_redirect_url` in `opensearch_dashboards.yml` inadvertently. The issues are solved now, but I opted to keep this in which waits for the logout API to finish before resuming the rest of this test which is a cleaner way of doing this since it avoids any race conditions.